magma/magma
# refactor(nms): Remove usages of tabs on organisations and user entities

## Summary

It used to be possible to configure accessible tabs for users and organisations. 
This no longer makes sense since all users have the same navigation.
All usages of the fields have been removed.

The fields will be removed from the entities later together with a migration. 

## Test Plan

- Make sure the navigation in existing organisations still works, also check for new users in existing organisations.
- Make sure the navigation work in new organisations.
- Make sure that rebuilding the docker container with an empty database still works.
- Make sure that organisations can be create via script.
